### DISCUSSION TITLE
inline code on light theme

### DISCUSSION THREAD

**Discussion by bernardolago** [hugo-sid/hugo-blog-awesome#154] [Q&A] [answered]
hi everyone

when using the dark theme, inline code works fine. when using the light theme, if as inline code is inside either a heading or a list, the colors don't change from default text color (text color and background), just the font changes. any thoughts?

**Comment by hugo-sid** [comment]
Hi @bernardolago. Thanks for using this theme.

Can you provide the following information?
- a screenshot which highlights your issue
- relevant Markdown content

**Comment by bernardolago** [comment]
hi @hugo-sid, thanks for checking in!

The first one is a paragraph and you can see that the background color for the inline code is displayed
![Screen Shot 2023-12-22 at 15 23 46](https://github.com/hugo-sid/hugo-blog-awesome/assets/57817217/cd024ca0-665e-469a-85af-9d9f59eca22c)

code:

```
### Commit

A commit is a kind of "snapshot" of the current state of the files in your repository. It is a way to record the changes you have made. Each commit has a descriptive message that explains the changes made. It is good practice to keep these messages concise but informative. For example, when using the command `git commit -m "Added chart functionality"`, you are recording a commit with the message "Added chart functionality" that reflects the changes you made to your files.
```

The second one is a numbered list
![Screen Shot 2023-12-22 at 15 24 26](https://github.com/hugo-sid/hugo-blog-awesome/assets/57817217/97e9ec15-e636-464f-a975-9fe336d7ca7c)

code:

```
### On Your Machine

1. Open the terminal and navigate to the folder where you want to create the repository.
2. Use the command `git init` to initialize a new local repository.
```

For reference, the background on the dark theme for the second example:
![Screen Shot 2023-12-22 at 15 27 15](https://github.com/hugo-sid/hugo-blog-awesome/assets/57817217/c4b39601-63f8-454e-aa63-5486f70c9d36)

website: bernardolago.com

**Reply by hugo-sid** [reply]
Hi @bernardolago. Thanks for providing the screenshots & Markdown content.

This is a bug.

**Reply by hugo-sid** [reply, marked as answer]
@bernardolago, I have created issue #158 regarding this bug.

## Fix 
 It should be fixed by PR #159.

FYI, the issue seems to be fixed in the demo site. You can check the following links & screenshot.

### Links

- Deploy preview link: https://deploy-preview-159--hugo-blog-awesome.netlify.app/posts/markdown-syntax/#list-types
- Corresponding Markdown file: https://github.com/hugo-sid/hugo-blog-awesome/blob/fix/code-in-list/exampleSite/content/en/posts/markdown-syntax/index.md#list-types

### Screenshot

![image](https://github.com/hugo-sid/hugo-blog-awesome/assets/122173059/1cf1c684-a996-445e-a596-07b73c7908f0)


Can you do some testing to let me know if this issue is fixed on your website?

**Reply by bernardolago** [reply]
Thanks!

**Reply by hugo-sid** [reply]
You're welcome @bernardolago !

I just merged the PR.

I suppose the way you are using this theme is as follows.

You have a fork of this repo/theme. You have added the fork as submodule.

If that is the case, please update/sync your fork to have this fix.

**Reply by hugo-sid** [reply]
@bernardolago, 

In case this issue is fixed in your website, please mark my comment ([this one](https://github.com/hugo-sid/hugo-blog-awesome/discussions/154#discussioncomment-7930428)) as answer.
In case the issue persists, don't hesitate to discuss more (or share more info).

**Reply by bernardolago** [reply]
That worked, thanks for that, this theme is the best
Cheers

**Reply by hugo-sid** [reply]
I enjoyed reading this https://bernardolago.com/posts/demystifying-regular-expressions-regex-in-python/ on your website :)

**Reply by bernardolago** [reply]
Thanks! That means a lot!
I've been pushing myself to create more content, it's been a fun challenge to come up with these articles 😄
